includes/Exception/SanitizationException.php

```diff
@@ -1,11 +1,11 @@
 <?php
 /**
- * Exception InvalidStopwatchEvent.
+ * Exception SanitizationException.
```

Comment:
Removed `InvalidStopwatchEvent` as it was never used in our plugin. Leftover from copying the original code I guess.